hwy/detect_targets.h

```diff
@@ -164,11 +164,13 @@
 #define HWY_BROKEN_CLANG6 0
 #endif
 
-// 32-bit may fail to compile AVX2/3.
+// 32-bit may fail to compile AVX2/3. GCC-13 was found ok
 #if HWY_ARCH_X86_32
-#define HWY_BROKEN_32BIT (HWY_AVX2 | (HWY_AVX2 - 1))
-#else
+#if (HWY_COMPILER_GCC_ACTUAL && HWY_COMPILER_GCC_ACTUAL >= 1300)
 #define HWY_BROKEN_32BIT 0
```

Comment:
HWY_AVX3 and newer are still going to cause trouble (see #2214), so we should change 0 to `(HWY_AVX3 | (HWY_AVX3 - 1))` for GCC13.